### ISSUE TITLE
Tutorial dialogs started popping up during mission

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#847]
After finishing the tutorial as an anonymous user, I was placed in a neighborhood as usual, but then after taking two steps, the tutorial started happening again, just on that street I was auditing... Best explained with pictures...

![sidewalk-why-label-again](https://user-images.githubusercontent.com/6518824/27990201-d71a4816-641c-11e7-9767-354cfa15f2d7.PNG)

I tried ignoring it for a bit to see if it would go away. Though I didn't take a picture, the "make sure to check both sides of the street" popup came up as it should have, and I dismissed it, but this remained.
![sidewalk-already-labeled-ramp](https://user-images.githubusercontent.com/6518824/27990193-c685b04e-641c-11e7-84ce-8618846a2e37.PNG)

Then eventually I followed the tutorial to see what would happen:
![sidewalk-phantom-ramp](https://user-images.githubusercontent.com/6518824/27990194-c7e813b4-641c-11e7-826f-49be876ad2b3.PNG)
![sidewalk-tutorial-follows-you](https://user-images.githubusercontent.com/6518824/27990196-cb4b93dc-641c-11e7-9f33-4096068bc47d.PNG)
![sidewalk-tutorial-keeps-going](https://user-images.githubusercontent.com/6518824/27990204-dd95a460-641c-11e7-9001-309500cc95cb.PNG)
Didn't take pics of all intermediate steps, but then...
![tutorial-finally-ends](https://user-images.githubusercontent.com/6518824/27990203-dd949f98-641c-11e7-9067-e40fe8a26317.PNG)

Put into a new neighborhood. (you may have noticed the red line going in both direction on the mini map, I will add a separate issue for that, they may or may not be related?)

**Comment by maddalihanumateja** [comment]
Could you provide more information so I can replicate this? So far I've been unsuccessful.

**Comment by misaugstad** [comment]
I have not been able to replicate it either. And I did not do anything out of the ordinary when this originally happened. The only thing that seemed different about this time was that the bug mentioned in #848 occurred as well.

**Comment by maddalihanumateja** [comment]
I got this same issue when I was testing the website out with a friend today. The user was dropped in  an intersection with the red line going in both direction. She zoomed in, labeled the entire intersection, got the "lets take a step" message, zoomed out, deleted a label and was placed in the tutorial. I was able to replicate this by following the same steps as her only once. The rest of my attempts have been unsuccessful. Unfortunately I didn't note the street edge id. 

<img width="1273" alt="screen shot 2017-07-09 at 1 02 38 pm" src="https://user-images.githubusercontent.com/3580069/27996201-380308d2-64ab-11e7-822b-0b5fdcf98a66.png">

**Comment by manaswis** [comment]
No reports of this issue occurring after the fix made in #850. Closing it for now. Can be reopened if the issue occurs again.

**Comment by maddalihanumateja** [comment]
One of turkers seems to have reported an issue which sounds exactly like this [on turkoption](https://turkopticon.ucsd.edu/reports?id=A2EFLXQOZNBUJW):

> "Initially I had a problem because after taking the tutorial once, it then somehow reappeared putting graphics and popups all over my first actual mission and I could not get it to go away. I finally had to reload the HIT which forced me to start over and do the first mission all over again. There is a feedback tab in the app so you can report stuff like that without having to email requester. After I re-loaded the HIT (and did not do the tutorial again obviously) I did not have the problem. "

We may need to open this issue again. It seems to be an infrequent bug.

**Comment by jonfroehlich** [comment]
Yes. Reopen.

**Comment by misaugstad** [comment]
Haven't seen this issue since 2017. We've made some big changes since then, so I'm thinking that this is likely fixed. Closing for now.
